catalog/core/catalog-core-localstorageprovider/src/test/java/org/codice/ddf/catalog/content/impl/FileSystemStorageProviderTest.java

```diff
@@ -171,7 +172,7 @@ public InputStream openStream() throws IOException {
                 mock(Metacard.class));
         UpdateStorageRequest updateRequest = new UpdateStorageRequestImpl(Collections.singletonList(
                 updateItem), null);
-
+        stream.close();
```

Comment:
Seems like a try-with-resources would be more appropriate then closing the stream here later. If anything goes wrong (e.g. exception thrown) here the stream.close() won't be called.